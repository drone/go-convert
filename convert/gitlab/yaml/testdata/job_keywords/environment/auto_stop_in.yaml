# https://docs.gitlab.com/ee/ci/yaml/#environmentauto_stop_in

review_app:
  script: deploy-review-app
  environment:
    name: review/$CI_COMMIT_REF_SLUG
    auto_stop_in: 1 day