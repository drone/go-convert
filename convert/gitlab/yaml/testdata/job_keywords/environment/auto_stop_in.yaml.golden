review_app:
    environment:
        name: review/$CI_COMMIT_REF_SLUG
        auto_stop_in: 1 day
    script:
        - deploy-review-app
